src/frontend/js/modules/sidebar.js

```diff
@@ -168,12 +183,63 @@ export default class Sidebar {
     this.nodes.sidebarContent.classList.toggle(Sidebar.CSS.sidebarContentHidden);
   }
 
+  /**
+   * Initializes sidebar
+   *
+   * @returns {void}
+   */
+  initSidebar() {
+    if (!this.isVisible) {
+      this.nodes.sidebar.classList.add(Sidebar.CSS.sidebarCollapsed);
+    }
+
+    /**
+     * prevent sidebar animation on page load
+     * Since animated class contains transition, hiding will be animated with it
+     * To prevent awkward animation when visibility is set to false, we need to remove animated class
+     */
+    setTimeout(() => {
+      this.nodes.sidebar.classList.add(Sidebar.CSS.sidebarAnimated);
+    }, 200);
+
+    // add event listener to execute keyboard shortcut
+    document.body.addEventListener('keydown', (e) => this.executeSlide(e), false);
+    document.body.addEventListener('keyup', (e) => this.executeSlide(e), false);
```

Comment:
You can use CodeX Shortcuts package for that

Reply:
@Y-k-Y I have added the support for period you can now use it